plugins/catalog-backend-module-github/src/providers/GitHubOrgEntityProvider.ts

```diff
@@ -88,6 +90,11 @@ export interface GitHubOrgEntityProviderOptions {
    * Optionally supply a custom credentials provider, replacing the default one.
    */
   githubCredentialsProvider?: GithubCredentialsProvider;
+
+  /**
+   * Optionally include a user mapper for mapping from GitHub users to UserEntities
+   */
+  userMapper?: IMapper<User, UserEntity>;
```

Comment:
Can we use the terminology "transformer" throughout instead of "mapper"? This is what it's called in other places. The comment in the LDAP one is

```ts
  /**
   * The function that transforms a user entry in LDAP to an entity.
   */
```

Reply:
done